docs/source/reference/feed_forward_configurator.rst

```diff
@@ -0,0 +1,50 @@
+Feed forward Configurators
+======
+
+Configurators are the way to perform a feed-forward computation.
+As they are non-unitary components, they can only be added to :code:`Processor` instances.
+
+Their purpose is to link measurements on given modes to circuits to configure.
+
+They are based on common principles:

```

Comment:
I would have said 'They are based on the following common features'. But this is your choice. When I see principles, my brain looks for some theory or rule for it.